### PR TITLE
Issue 380: Upgrade alpine image to fix security vulnerabilities 

### LINKED ISSUE #380: Upgrade alpine image to fix security vulnerabilities
### Description

Currently zookeeper-operator is using `alpine:3.11` and there are some security vulnerabilities reported 

### Importance

should-have

### Location

Dockerfile

### Suggestions for an improvement

Upgrade alpine image

### PR DESCRIPTION
**Pull request by anishakj**
Signed-off-by: anishakj <anisha.kj@dell.com>

### Change log description

Upgraded alpine image to `3.14`

### Purpose of the change

 Fixes #380

### What the code does

Upgraded alpine image to `3.14`

### How to verify it

All e2e should pass

### CODE REVIEW

**Review by SrishT — APPROVED** [review]
LGTM

**Review by gaddamas — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ARG DOCKER_REGISTRY
-ARG GO_VERSION=1.13.8
-ARG ALPINE_VERSION=3.11
-FROM ${DOCKER_REGISTRY:+$DOCKER_REGISTRY/}golang:${GO_VERSION}-alpine${ALPINE_VERSION} as go-builder
+ARG ALPINE_VERSION=3.14
+FROM ${DOCKER_REGISTRY:+$DOCKER_REGISTRY/}golang:1.13.8-alpine3.11 as go-builder
```
[the remark this answers is not in the document]
Comment (anishakj) [reply Dockerfile:3]: yes, `golang:1.13.8-alpine3.14` is not available in dockerhub.

[the remark this answers is not in the document]
Comment (amuraru) [reply Dockerfile:3]: @anishakj why not using golang 1.16 here?

[the remark this answers is not in the document]
Comment (anishakj) [reply Dockerfile:3]: `golang:1.16-alpine3.14` was giving compilation errors

**Pull request merged**